pylint_odoo/misc.py

```diff
@@ -112,14 +113,27 @@ def get_manifest_file(self, node_file):
 
     def set_ext_files(self):
         """Create `self.ext_files` dictionary with {extension_file: [files]}
+            and exclude files using --ignore and --ignore-patterns parameters
         """
         self.ext_files = {}
         for root, _, filenames in os.walk(self.module_path, followlinks=True):
             for filename in filenames:
+                find = True
                 fext = os.path.splitext(filename)[1].lower()
                 fname_rel = os.path.relpath(
                     os.path.join(root, filename), self.module_path)
-                self.ext_files.setdefault(fext, []).append(fname_rel)
+                name = os.path.join(self.module_path, fname_rel)
```

Comment:
Add a pdb and print the value `os.path.join(root, filename)` and `name` and tell me what is the difference